ruma-api/Cargo.toml

```diff
@@ -15,6 +15,9 @@ repository = "https://github.com/ruma/ruma"
 version = "0.17.0-alpha.1"
 edition = "2018"
 
+[features]
+unstable-synapse-quirks = ["ruma-events/unstable-synapse-quirks"]
```

Comment:
Since ruma-events is only a dev-dependency for ruma-api, I don't this this feature should exist here.